### PR TITLE
Establish baseline website for 2020

### PR DESCRIPTION
**Pull request by theacodes**

### CODE REVIEW

**Review by EstiShay — APPROVED** [review]

File: content/code-of-conduct-reporting/contents.lr [hunk 1]
```diff
@@ -29,67 +29,19 @@ those individuals instruct us otherwise. In the event of involvement with law en
 If you believe anyone is in physical danger, please notify appropriate law
 enforcement first. In the United States, emergency services can be contacted by
 dialing 911. The non-emergency line for the Seattle Police Department is
```
Comment (roadsideseb) [inline content/code-of-conduct-reporting/contents.lr]:
```suggestion
dialing 911. The non-emergency line for the Portland Police Department is
```

File: content/grants/contents.lr [hunk 2]
```diff
@@ -9,18 +9,9 @@ Grants
 
 Our grants program is an important tool that helps us realize a goal of creating an inclusive, respectful conference that welcomes people of all races, ethnicities, genders, ages, abilities, religions, and sexual orientations. We provide financial assistance so people who might otherwise not afford to join our event can. 
 
-In our first year 2018, we dedicated a large portion of our budget for grants. For our upcoming conference in 2019 we intend to greatly increase grants awarded. Grants recipients can have some part of their expenses (which can include travel, hotel, registration, and childcare) paid from PyCascades’s budget. The organizers of the conference actively encourage people to apply for financial aid. 
+During our first two years we dedicated a large portion of our budget for grants. For our upcoming conference in 202 we intend to once again increase grants awarded. Grants recipients can have some part of their expenses (which can include travel, hotel, registration, and childcare) paid from PyCascades’s budget. The organizers of the conference actively encourage people to apply for financial aid.
```
Comment (roadsideseb) [inline content/grants/contents.lr]:
```suggestion
During our first two years we dedicated a large portion of our budget for grants. For our upcoming conference in 2020 we intend to once again increase grants awarded. Grants recipients can have some part of their expenses (which can include travel, hotel, registration, and childcare) paid from PyCascades’s budget. The organizers of the conference actively encourage people to apply for financial aid. 
```

File: content/news/1-announcing-pycascades-2020/contents.lr [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+pleasure to announce that **Eric Holscher** and **Esti Shay** will be leading
+PyCascades 2020 as our co-chairs. You can read more about them on our
+`organizers </organizers>`_ page or say hi 👋 to them
+`on Slack <http://bit.ly/pycascades-slack>`_.
+
+We want to put on an amazing show and make sure it's possible for all
+people to attend. We're committed to continuing and expanding our opportunity
+grants program. To make all this happen we're seeking
+`sponsors </become-a-sponsor>`_. Please reach out to us at
+organizers@pycascades.com if you're interested.
```
Comment (roadsideseb) [inline content/news/1-announcing-pycascades-2020/contents.lr]:
```suggestion
sponsorship@pycascades.com if you're interested.
```

File: content/volunteer/contents.lr [hunk 4]
```diff
@@ -14,7 +14,7 @@ Each year we ask our attendees to get involved and help us make this event succe
 Sign up
 +++++++
 
-If you're interested in volunteering, please take a moment to read the brief descriptions below and fill out the `volunteer sign up form <https://bit.ly/pycascades-volunteers-2019>`__. Reach out to organizers@pycascades.com if you have any questions or concerns.
+If you're interested in volunteering, that's wonderful! We're not quite ready to sign up volunteers for 2020 but feel free to reach out to organizers@pycascades.com if you'd like to chat or have questions in the meantime.
```
Comment (roadsideseb) [inline content/volunteer/contents.lr]:
```suggestion
If you're interested in volunteering, that's wonderful! We're not quite ready to sign up volunteers for 2020 but feel free to reach out to volunteers@pycascades.com if you'd like to chat or have questions in the meantime.
```

File: templates/base.jade [hunk 5]
```diff
@@ -3,7 +3,7 @@ doctype html
 html
 
     head
-        title {% block page_title %}PyCascades 2019 &ndash; Seattle, WA{% endblock %}
+        title {% block page_title %}PyCascades 2020 &ndash; Portland, Oregan{% endblock %}
```
Comment (roadsideseb) [inline templates/base.jade]:
```suggestion
        title {% block page_title %}PyCascades 2020 &ndash; Portland, Oregon{% endblock %}
```

**Review by roadsideseb — APPROVED** [review]
Just found a few typos. I'll correct those and go ahead and merge it.

**Comment by ericholscher** [comment]
🎉

**Pull request merged**